package.json

```diff
@@ -5,8 +5,11 @@
     "main": "dist/oidc-client.js",
     "scripts": {
         "start": "node sample/server.js",
-        "build": "webpack --config webpack.config.js",
-        "test": "mocha --compilers js:babel-register test/unit/*.spec.js"
+        "build": "webpack -p --config webpack.config.js",
+        "build:classic": "webpack -p --config webpack.config.classic.js",
+        "prebuild": "npm run build:classic",
+        "test": "mocha --compilers js:babel-register test/unit/*.spec.js",
+        "pretest": "npm run build"
```

Comment:
This is just a convenience method to make sure that the build we test on is always up to date. Just delete it if you don't want it.